### PR TITLE
NM: Get 2017 working

### PR DESCRIPTION
**Pull request by mileswwatkins**
Cherry-picked for the state, as part of #1170

### CODE REVIEW

File: openstates/nm/bills.py [hunk 1]
```diff
@@ -97,8 +97,10 @@ class NMBillScraper(BillScraper):
     categorizer = Categorizer()
 
     def _init_mdb(self, session):
+        print session[2:]
         ftp_base = 'ftp://www.nmlegis.gov/other/'
-        fname = 'LegInfo{}'.format(session[2:])
+        #session slug here?
+        fname = 'LegInfo{}'.format('16 Special')
```
Comment (showerst) [inline openstates/nm/bills.py]: This hack is bad, and i should feel bad. 

TODO: Figure out the right way to format this that works across sessions.

**Comment by mileswwatkins** [comment]
Legislators still are 2016ers; wait for [Baca](https://ballotpedia.org/Gregory_A._Baca) to replace Sanchez in Senate 29.

**Comment by mileswwatkins** [comment]
Looks like Senate listing has been updated. Senate committees have been wiped to have no members. House committees have member names, but I don't have a chance right now to check if they're current or old. But NM should be almost ready to go, after a final data quality check.

**Comment by mileswwatkins** [comment]
@showerst, if you run this and it looks good, feel free to merge this branch in.

**Pull request merged**